### PR TITLE
Update wrr_traffic_test.go 

### PR DESCRIPTION
**Pull request by saarvind6481**
This PR is for changing weights for cisco in accordance with https://github.com/openconfig/featureprofiles/pull/1242/

### CODE REVIEW

File: feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go [hunk 1]
```diff
@@ -1714,7 +1509,7 @@ func ConfigureCiscoQos(t *testing.T, dut *ondatra.DUTDevice) {
 		targetGrpoup string
 	}{{
 		desc:         "scheduler-policy-BE1",
-		sequence:     uint32(6),
+		sequence:     uint32(1),
```
Comment (guoshiuan) [inline feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go:1512]: Can we define human-friendly constants for these magic seq numbers: 0, 1, 4, 16?

Comment (saarvind6481) [reply feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go:1512]: We can have , but general convention followed is for priority queues we use zero and one for non priority queues

**Review by guoshiuan — APPROVED** [review]

**Review by rszarecki — APPROVED** [review]

File: feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go (hunk 1 above)
Comment (rszarecki) [reply feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go:1510]: The OpenConfig MANDATES that sequence 0 is served 1st. And only when queue is empty (or PIR is achived) secheduler 1 is served. And so on.

So it is not convention. It's "standard".

**Pull request merged**